code/modules/mob/mob_defines.dm

```diff
@@ -27,6 +27,7 @@
 	var/track_blood = 0
 	var/feet_blood_color
 	var/datum/skills/skills
+	var/shoot_inactive_hand = FALSE
```

Comment:
need autodoc
///In akimbo firing, if the next hand shooting will be the inactive hand

Reply:
Autodoc?